src/main/java/com/teragrep/rlp_03/context/buffer/BufferLeaseImpl.java

```diff
@@ -28,13 +29,8 @@ public long id() {
 
     @Override
     public long refs() {
```

Comment:
this should be removed or at least marked as unreliable because phaser is a lock free structure so ref count should be only used for debug logging